packages/@glimmer/runtime/lib/syntax/functions.ts

```diff
@@ -280,7 +285,14 @@ export class PartialInvoker implements DynamicInvoker<ProgramSymbolTable> {
     for (let i = 0; i < partialSymbols.length; i++) {
       let name = partialSymbols[i];
       let symbol = i + 1;
-      let value = evalScope[name];
+
+      let value;
+      //TEMP: GJ: get the tests passing, we'll likely need to do something like merge scope and then check
+      if(evalScope) {
+        value = evalScope[name];
+      } else {
+        value = locals[name];
+      }
```

Comment:
I'm unclear on the differences between `locals` and `evalScope`. Here's some questions:

1. should `locals` contain the `outerScope.getPartialMap()` like I'm doing above or perhaps it belongs in `evalScope`?
2. when iterating over the `partialSymbols`, should we alway be checking `evalScope`?

Reply:
locals are supposed to win, they are names the partial introduced.

Reply:
thanks